payment/controllers_test.go

```diff
@@ -418,7 +418,8 @@ func (suite *ControllersTestSuite) TestAnonymousCardE2E() {
 	createRequest := &CreateOrderRequest{
 		Items: []OrderItemRequest{
 			{
-				SKU:      "MDAxN2xvY2F0aW9uIGJyYXZlLmNvbQowMDFhaWRlbnRpZmllciBwdWJsaWMga2V5CjAwMzJjaWQgaWQgPSA1Yzg0NmRhMS04M2NkLTRlMTUtOThkZC04ZTE0N2E1NmI2ZmEKMDAxN2NpZCBjdXJyZW5jeSA9IEJBVAowMDE1Y2lkIHByaWNlID0gMC4yNQowMDJmc2lnbmF0dXJlICRlYyTuJdmlRFuPJ5XFQXjzHFZCLTek0yQ3Yc8JUKC0Cg",
+				SKU: "AgEJYnJhdmUuY29tAgpwdWJsaWMga2V5AAInaWQ9NWM4NDZkYTEtODNjZC00ZTE1LTk4ZGQtOGUxNDdhNTZiNmZhAAISc2t1PWFub24tY2FyZC12b3RlAAIMY3VycmVuY3k9QkFUAAIKcHJpY2U9MC4yNQAABiCBp8pJJYFZwJC7w2HjT-Sb6ogHOw-BnhLORRtGH36bhQ",
+				//SKU:      "MDAxN2xvY2F0aW9uIGJyYXZlLmNvbQowMDFhaWRlbnRpZmllciBwdWJsaWMga2V5CjAwMzJjaWQgaWQgPSA1Yzg0NmRhMS04M2NkLTRlMTUtOThkZC04ZTE0N2E1NmI2ZmEKMDAxN2NpZCBjdXJyZW5jeSA9IEJBVAowMDE1Y2lkIHByaWNlID0gMC4yNQowMDJmc2lnbmF0dXJlICRlYyTuJdmlRFuPJ5XFQXjzHFZCLTek0yQ3Yc8JUKC0Cg",
```

Comment:
nit: can we delete commented SKU?

Reply:
incorporated: https://github.com/brave-intl/bat-go/pull/303/commits/deffea037791db48c2631fe6ff4c5167c677ac49